### ISSUE TITLE
[FEATURE REQUEST]: Allow automatic export after compilation

### ISSUE THREAD

**Issue by Dotneteer** [Dotneteer/spectnetide#189]
In the discussion about SpectNetIDE-ZEsarUX integration (with Mike van der Lee), it came to light that an automatic export of the compilation could be a useful solution.

Here is a private build of SpectNetIDE, which contains this feature:

[Spect.Net.VsPackage.zip](https://github.com/Dotneteer/spectnetide/files/4496157/Spect.Net.VsPackage.zip)

(Do not forget to rename the file to `.vsix` before install.)

You can find the documentation of this new feature here: https://dotneteer.github.io/spectnetide/getting-started/export-a-z80-program-2#auto-export

**Comment by TheStruggleForAntiSpaghetti** [comment]
I've set-up this built and created a simple ZX Basic project,.

When trying to run it, I'm running into compiling errors (see included screenshot in zip).
 
See the project folder attached.

The batch-file I'm calling in the BuildTasks is the one calling ZEsarUX in the ZEsarUX folder I shared the other day.

[SpectnetIDE-Dev_esxDOS-Dir.zip](https://github.com/Dotneteer/spectnetide/files/4496859/SpectnetIDE-Dev_esxDOS-Dir.zip)

**Comment by Dotneteer** [comment]
Mike, today I examined the project you sent me. The issue with the code is that the the `exdos.bas` file contains the `local`, `proc`, and `endp` assembly keywords with uppercase first letters: `Local`, `Proc`, and `Endp`.  Thus, the Assembler takes these words into account as label names, and not as statements. SpectNetIDE accepts either full lowercase or full uppercate keywords, so `LOCAL`, `PROC`, and `ENDP` are also valid.
I already had had to add some tweaks to the SpectNetIDE assembler to handle ZX BASIC-specific issues that caused incompatibilities, the one you found is just a new of them :-).
I try to find out a tweak to solve this issue. I examine the files you sent me further, I guess there might be still other issues with them hidden by the wrong keywords. As soon as possible, I will send you a new private build for testing.

**Comment by Dotneteer** [comment]
Mike, here is the private build I mentioned. I found, the issue was only with the `Local` keyword, earlier I already solved the issue with `Proc` and `Endp`. This private build works on my machine, but if you uses a different ZXB compiler than I, it may raise issues. Anyhow, just send me the zipped project code, and I will hunt down the other incompatibility issues, too.

[Spect.Net.VsPackage.zip](https://github.com/Dotneteer/spectnetide/files/4527081/Spect.Net.VsPackage.zip)
